_posts/2016-01-25-drupal-6-part-3.md

```diff
@@ -0,0 +1,166 @@
+
+This is of course meant to err on the side of caution, but can 
+also be very anxiety provoking. Similarly, when hearing of the 3 month window for Drupal 6 
+support after Drupal 8.0.0 was released ["Le dendrite"](https://www.drupal.org/drupal-6-eol#comment-10556816) is not wrong to say:
+
+> someone please settle my stomach
+>
+>  I was shocked to read this. it might seriously wreck me,
+>  i really hope i'm misunderstanding this
+>  
+>  will my sites just die?
+
+That [entire thread](https://www.drupal.org/drupal-6-eol) is a 
+pretty good exposé into the various perspectives and preparedness
+on upgrading Drupal 6 sites. If you've got a spare half-day, give it a once-over.
+
+On the **_other hand_**, we have had at least two clients in 2015 who continued to run 
+[**Drupal 5**](https://www.drupal.org/drupal-5.0) (not a typo) 
+websites in production successfully. 
+
+Neither end is entirely wrong, they just represent difference of opinion and available options.
+
+
+### The primary options for Drupal 6 site owners are
+1. Do **not** upgrade your website to a newer Drupal version
+1. **Do** upgrade your website to a newer Drupal version
+1. Use the time to seriously revision your website altogether
+
+But how do you decide which option is best for you, the site owner?
+
+That decision mostly rests on your organization's
+ 
+ + risk tolerance
+ + likelihood to be targeted
+ + budget to improve your website
+
+
+##### Considerations to _not_ upgrade
+
+Choosing _not_ to upgrade your website is typically dictated by your organization having neither internal
+technical expertise or sufficient funds to hire a partner. When this is the case, if a small budget can be
+afforded, a site audit is usually the next best option to have some peace of mind that you're taking the measures your're able to
+ mitigate the risk of running outdated software. A group like Savas Labs can perform a 
+thorough assessment of risk and make budget-conscious recommendations
+on how to best harden the site against outside threats without doing the more costly
+full-site upgrade. Primary attack vectors that we focus lie in custom code, and 
+under-supported contributed modules since core code represents much lower risk
+having been well vetted by years in the wild.
+
+ <img src="/assets/img/wild.jpg" alt="in the wild">
+
+One other consideration independent
+of your specific website configuration is your server autonomy dictated by your hosting provider. Running old software is
+usually tolerated in a shared hosting environment for only so long before upgrades
+are enforced. Warning emails are often (not always) sent out in advance of server-level upgrades, however,
+ it's easy to miss those emails, and it hurts to be caught by surprise that some PHP version upgrade renders
+your site completely useless. Understanding the level of control you as the 
+site owner has over your server is crucial.
+
+    
+##### Considerations in upgrading
```

Comment:
Consider making this phrase "Considerations in upgrading" similar to the title above "Considerations to not upgrade". Maybe change the above phrase to "Considerations in not upgrading" @chrisarusso